executor/runtime/container.go

```diff
@@ -51,6 +55,12 @@ func NewContainer(taskID string, titusInfo *titus.ContainerInfo, resources *runt
 	labels[diskLabelKey] = strDisk
 	labels[networkLabelKey] = strNetwork
 
+	passthroughAttributes := titusInfo.GetPassthroughAttributes()
+	if passthroughAttributes != nil {
+		labels[ownerEmailLabelKey] = passthroughAttributes[ownerPassThroughKey]
+		labels[jobTypeLabelKey] = passthroughAttributes[jobTypePassThroughKey]
```

Comment:
It's my understanding that if a value isn't present in the map you get the zero value which for strings is the empty string, which for me is desired behavior.